tests/OpenCloud/Tests/Database/Resource/InstanceTest.php

```diff
@@ -91,4 +91,9 @@ public function testUserList()
     {
         $this->assertInstanceOf(self::COLLECTION_CLASS, $this->instance->userList());
     }
+
+    public function testBackupList()
+    {
```

Comment:
can we also add a separate test for creating a DB instance with a restore point?